salt/states/file.py

```diff
@@ -663,13 +663,13 @@ def _error(ret, err_msg):
 
 
 def _check_directory(name,
-                     user,
-                     group,
-                     recurse,
-                     mode,
-                     clean,
-                     require,
-                     exclude_pat,
+                     user=None,
```

Comment:
This scares me a little bit. Why is it necessary to set all the defaults suddenly?

Reply:
There are two instances where this function is called. It only specifies the name of the directory. All other values are hard values (no variables). Those are here:
- https://github.com/saltstack/salt/pull/47533/files#diff-3b447dc835943beb1326f98f684fe783L4706
  and here:
- https://github.com/saltstack/salt/pull/47533/files#diff-3b447dc835943beb1326f98f684fe783L4899
Since both calls to this function were passing the exact same things, I made them the defaults. This allows you to only pass the items you wish to check instead of having to pass all options every time.